dashboard/prisma/schema.prisma

```diff
@@ -288,6 +290,33 @@ model BugReport {
   @@index([userId])
 }
 
+model Annotation {
+  id String @id @default(cuid())
+
+  dashboard Dashboard @relation(fields: [dashboardId], references: [id], onDelete: Cascade)
+  dashboardId String
+
+  // Chart identifier (e.g., "overview", "web-vitals", "page-traffic", etc.)
+  chartId String
+
+  date DateTime
+  label String @db.VarChar(48)
+  description String? @db.VarChar(256)
+
+  colorToken String?
+
+  createdBy User @relation(fields: [createdById], references: [id], onDelete: Cascade)
+  createdById String
```

Comment:
Is there a specific reason as to why we want the relation to the user who created this? 🤔